package.json

```diff
@@ -21,23 +21,24 @@
     "url": "https://github.com/andrewMacmurray/seeds-game/issues"
   },
   "homepage": "https://github.com/andrewMacmurray/seeds-game#readme",
-  "dependencies": {
-    "bounce.js": "^0.8.2",
-    "elm": "^0.19.0",
-    "howler": "^2.0.9",
-    "pepjs": "^0.4.3",
-    "tachyons-custom": "^4.5.4"
-  },
   "devDependencies": {
+    "@types/howler": "^2.1.1",
     "autoprefixer": "^6.7.3",
+    "bounce.js": "^0.8.2",
+    "elm": "^0.19.0",
     "elm-hot": "^1.1.2",
     "elm-live": "^3.0.6",
+    "elm-typescript-interop": "0.0.15",
```

Comment:
is your plan to migrate all your `.js` to `.ts` ?

Reply:
I think there's only 1 `.js` file left which is config `postcss-config.js`.